addon/helpers/sort-option-display.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+
+/**
```

Comment:
Verify that this file shows up correctly in built docs. It may need the `@module` annotation.